buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+plugins {
+    `kotlin-dsl`
+}
+
+dependencies {
+    "implementation"("com.android.tools.build:gradle:3.2.1")
```

Comment:
is there a way to inherit the gradle version from the root `build.gradle` file or is it ok to leave this with a custom version?  (Would we need to bump the gradle version in two places?)

For example, see https://github.com/mozilla/application-services/pull/921/files

Reply:
Done!